apps/backend/src/routers/api/admin/divisions/users.ts

```diff
@@ -15,7 +15,13 @@ router.get('/', (req: Request, res: Response) => {
 router.get(
   '/export',
   asyncHandler(async (req: Request, res: Response) => {
-    const users = await db.getDivisionUsersWithCredentials(new ObjectId(req.params.divisionId));
+    const divisionUsers = await db.getDivisionUsersWithCredentials(
+      new ObjectId(req.params.divisionId)
+    );
+    const eventUsers = await db.getUsersWithCredentials({
+      assignedDivisions: new ObjectId(req.params.divisionId)
```

Comment:
Assigned divisions is an array - should the filter be a contains function?

Reply:
Turns out that mongodb doesn't have a $include function for querying, yet this syntax does is a shorthand for the same thing, checking if assignedDivisions includes the relevant value. Should we maybe add a comment explaining this?

Reply:
Standard mongo knowledge so no need